pkgs/by-name/at/atlantis/package.nix

```diff
@@ -27,6 +30,16 @@ buildGoModule (finalAttrs: {
 
   doInstallCheck = true;
   nativeInstallCheckInputs = [ versionCheckHook ];
+  nativeBuildInputs = [ makeWrapper ];
+  postFixup = ''
+    wrapProgram $out/bin/atlantis \
+      --prefix PATH : ${
+        lib.makeBinPath [
+          bash
+          git
```

Comment:
Should the terraform client be added here as well?

https://github.com/runatlantis/atlantis/blob/94dfea4184348e6f793b2dfe05d5d81f79182a27/server/core/terraform/tfclient/terraform_client.go#L586

Reply:
Terraform version is managed by atlantis. Multiple projects can use different terraform versions.

https://www.runatlantis.io/docs/terraform-versions